content/posts/customizing/customize-layout/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Customizing the layout"
+date: 2021-08-07T06:20:50+06:00
+hero: /images/posts/configuration/about-section-hero.svg
+author:
+  name: James Ray
+  image: images/author/james.jpg
+menu:
+  sidebar:
+    name: Layout Customization
+    identifier: customizing-layout
+    parent: customizing
+    weight: 411
+---
+
+## We've all been there
+We all want to be able to customize the layout of a given theme, and I with this short tutorial I will show you how. I stumbled my way through this, and I hope you won't have to as well.
```

Comment:
Please, change `and I with this` to `and in this`

Reply:
Thank you for that catch!